pom.xml

```diff
@@ -61,15 +61,15 @@
         <maven.compiler.release>17</maven.compiler.release>
         <maven.compiler.useIncrementalCompilation>false</maven.compiler.useIncrementalCompilation>
         <maven.version>3.9.6</maven.version>
-        <protoc.version>3.17.3</protoc.version>
+        <protoc.version>4.28.2</protoc.version>
```

Comment:
May I ask why you propose this old version instead of the latest, @mgabelle ?
- https://mvnrepository.com/artifact/com.google.protobuf/protobuf-java/4.29.3